<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>672</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ravie</family>
    <pointsize>28</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.119318 rgba(0, 255, 255, 255), stop:0.522727 rgba(57, 252, 255, 255), stop:0.778409 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>319</y>
     <width>1310</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);
color: rgb(255, 85, 127);
font: 18pt &quot;Forte&quot;;</string>
   </property>
   <property name="text">
    <string>Player  1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Forte&quot;;
color: rgb(255, 85, 127);
background-color: rgb(124, 253, 255);</string>
   </property>
   <property name="text">
    <string>Player  2</string>
   </property>
  </widget>
  <widget class="QLabel" name="pl1text">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 255, 255);
color: rgb(85, 85, 255);
font: 20pt &quot;Forte&quot;;</string>
   </property>
   <property name="text">
    <string>Enter the word you wanna challenge with ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="pl2text">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(137, 253, 255);
color: rgb(53, 53, 255);
font: 20pt &quot;Forte&quot;;
background-color: qlineargradient(spread:pad, x1:0.500318, y1:0.568, x2:0.511, y2:1, stop:0.625 rgba(140, 253, 255, 255), stop:1 rgba(167, 254, 255, 255));</string>
   </property>
   <property name="text">
    <string>Enter the word you wanna challenge with ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="pl1turn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>arrow_label</string>
   </property>
  </widget>
  <widget class="QLabel" name="pl2turn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>arrow_label</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pl1word">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 252, 147);
font: 18pt &quot;Forte&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pl2word">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 252, 147);
font: 18pt &quot;Forte&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pl1disp">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(20, 254, 255);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>C O U N T D O _ N</string>
   </property>
  </widget>
  <widget class="QLabel" name="pl2disp">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>480</y>
     <width>541</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0.46, y1:0, x2:0.551, y2:1, stop:0 rgba(208, 254, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>C O U N T D O _ N</string>
   </property>
  </widget>
  <widget class="QLabel" name="try1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 253, 255);
color: rgb(64, 64, 255);</string>
   </property>
   <property name="text">
    <string>Try with a letter to solve the word :</string>
   </property>
  </widget>
  <widget class="QLabel" name="try2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>590</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(53, 53, 255);</string>
   </property>
   <property name="text">
    <string>Try with a letter to solve the word :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="char1">
   <property name="geometry">
    <rect>
     <x>1115</x>
     <y>259</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 255, 255);
color: rgb(64, 64, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="char2">
   <property name="geometry">
    <rect>
     <x>1115</x>
     <y>599</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(207, 255, 255);
color: rgb(53, 53, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pl1level">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 254, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="pl2level">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.46, y1:0, x2:0.5, y2:1, stop:0 rgba(167, 254, 255, 255), stop:0.420455 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
